client/components/data/query-reader-related-posts/index.jsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { shouldFetchRelated } from 'state/reader/related/selectors';
```

Comment:
Should this part of the Redux tree be named `state/reader/related-posts` to be clear?